src/main/java/org/opensearch/knn/index/engine/LuceneHNSWContext.java

```diff
@@ -1,18 +1,11 @@
 /*
+ * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
  */
 
-package org.opensearch.knn.index.util;
+package org.opensearch.knn.index.engine;
 
 import com.google.common.collect.ImmutableMap;
-import org.opensearch.knn.index.Parameter;
 import org.opensearch.knn.index.query.request.MethodParameter;
 
```

Comment:
This file needs to be under lucene if we are going with one package per engine

Reply:
Makes sense. Will move